### DISCUSSION TITLE
SOC Cases export to PDF.

### DISCUSSION THREAD

**Discussion by EastStriker** [Security-Onion-Solutions/securityonion#8790] [Ideas] [answered]
Hello,

I would like to reach out and ask if it may be possible to have a feature for Cases within the Security Onion Console where analysts are able to export documented reports of interests to PDF for reporting purposes? I do believe since Hive used to have a similar capability when it was previously integrated with Security Onion.

If there is a possible approach to Cases exported to PDF, would there be a capability to customizing templates and adding illustrations (such as screenshots)?

Much appreciated.

**Comment by dougburks** [comment, marked as answer]
We might build a Cases PDF export feature in the future, but I can't make any promises right now.

You might be able to achieve your goal using something like Puppeteer:
https://pptr.dev/
